dev/src/DocFx/Command/DocFx.php

```diff
@@ -42,6 +42,7 @@ protected function configure()
             ->addOption('out', '', InputOption::VALUE_REQUIRED, 'Path where to store the generated output.', 'out')
             ->addOption('metadata-version', '', InputOption::VALUE_REQUIRED, 'version to write to docs.metadata using docuploader')
             ->addOption('staging-bucket', '', InputOption::VALUE_REQUIRED, 'Upload to the specified staging bucket using docuploader.')
+            ->addOption('component-path', '', InputOption::VALUE_OPTIONAL, 'Specify the path of the desired component.')
```

Comment:
I think it would be clearer to make this `snippet-path`. This would help differentiate it from the existing `Component` option, and also make the variable passed into `PageTree` more descriptive.

Reply:
This is a little tricky. It is used to derive some other content that isn't snippet related, like from the README/repo-metadata in the component directory. This is only really here to help facilitate testing. What are your thoughts on me adding that to the description? Something like `Please note, this option is only intended for testing purposes`.

Reply:
That works for me

Reply:
Updated!